libs/wingc/src/ast.rs

```diff
@@ -657,31 +664,31 @@ impl Display for Reference {
 }
 
 /// Represents any type that has a span.
-pub trait ToSpan {
+pub trait Locatable {
```

Comment:
A [very common](https://github.com/search?q=Spanned+lang%3ARust+&type=code) name for is trait is `Spanned`

Reply:
done